site/src/js/index.js

```diff
@@ -1,103 +1,106 @@
-/* global Selectr */
+/* global window, document, Selectr */
 
 /**
  * Initialising global variables
  */
 const ACTIVE_CLASS = 'visible';
 const sort = {
-    ALPHABETICAL_ASCENDING: (a,b) => a.dataset.name > b.dataset.name ? 1 : -1,
-    ALPHABETICAL_DESCENDING: (a,b) => a.dataset.name < b.dataset.name ? 1 : -1,
-    DIFFICULTY_ASCENDING: (a,b) => a.dataset.difficulty > b.dataset.difficulty ? 1 : -1,
-    DIFFICULTY_DESCENDING: (a,b) => a.dataset.difficulty < b.dataset.difficulty ? 1 : -1
+  ALPHABETICAL_ASCENDING: (a, b) => a.dataset.name > b.dataset.name ? 1 : -1,
+  ALPHABETICAL_DESCENDING: (a, b) => a.dataset.name < b.dataset.name ? 1 : -1,
+  DIFFICULTY_ASCENDING: (a, b) => a.dataset.difficulty > b.dataset.difficulty ? 1 : -1,
+  DIFFICULTY_DESCENDING: (a, b) => a.dataset.difficulty < b.dataset.difficulty ? 1 : -1
 };
 
 const contentEl = document.getElementById('content');
 const filterInput = document.getElementById('filter');
 const sortingInput = document.getElementById('sorting');
 
-const activateElements = els => Array.from(els).forEach(node => node.classList.add(ACTIVE_CLASS));
+const activateElements = els => [...els].forEach(node => node.classList.add(ACTIVE_CLASS));
```

Comment:
I'd prefer if we don't spread for clientside code as transpilation adds a proxy function for this, which unnecessarily increases the bundle size

Reply:
it's from the default unicorn rules: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prefer-spread.md

Reply:
Yeah, in this case I think it would be better to override this rule for the client js 🤔

Reply:
rule disabled with `eslint-disable`

Here's the config for reference
```
  "xo": {
    "overrides": [
      {
        "files": "src/js/**/*.js",
        "rules": {
          "unicorn/prefer-spread": "off"
        }
      }
    ]
  },
```